amundsen_application/static/css/_buttons-default.scss

```diff
@@ -28,6 +28,37 @@
     margin-bottom: 4px;
   }
 
+  &.btn-cancel {
+    border-color: $gray-lighter;
+    font-family: $font-family-sans-serif-bold;
+    font-weight: $font-weight-sans-serif-bold;
+
+    &:focus,
+    &:not(.disabled):hover,
+    &:not([disabled]):hover {
+      background-color: $gray-lighter;
+      border-color: $gray-lighter;
+    }
+  }
+
+  &.btn-icon {
```

Comment:
I think the class name should indicate that this is a flat button with no border. We already have some buttons that have icons inside.

Reply:
`btn-flat-icon`?